ci/jenkins/unity_jenkinsfile.groovy

```diff
@@ -318,6 +318,7 @@ stage('Build and Test') {
           sh "${docker_run}  ${ci_gpu} ./tests/scripts/task_config_build_gpu.sh build"
           make("${ci_gpu}", 'build', '-j2')
           sh "${docker_run} ${ci_gpu} ./tests/scripts/unity/task_python_relax_gpuonly.sh"
+          sh "${docker_run} ${ci_gpu} ./tests/scripts/unity/task_python_msc.sh"
```

Comment:
It appears that the test cases for your MSC were not executed as indicated by the following link: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-unity/detail/PR-15500/7/pipeline/60/. The changes to the Jenkins file will only take effect once they are merged. To run your test cases on CI machines for this pull request, you might consider adding them to the test_python_relax.py or test_python_relax_gpu.py files.

Reply:
I add test_msc to task_python_relax.sh. Seems ok: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-unity/detail/PR-15500/8/pipeline/60

Reply:
yep, the tests work in CI.